Converters/Rhino/Speckle.Converters.RhinoShared/ToHost/Raw/RegionToHostConverter.cs

```diff
@@ -0,0 +1,40 @@
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Objects;
+
+namespace Speckle.Converters.Rhino.ToHost.Raw;
+
+public class RegionToHostConverter : ITypedConverter<SOG.Region, RG.Hatch>
+{
+  private readonly ITypedConverter<ICurve, RG.Curve> _curveConverter;
+  private readonly IConverterSettingsStore<RhinoConversionSettings> _settingsStore;
+
+  public RegionToHostConverter(
+    IConverterSettingsStore<RhinoConversionSettings> settingsStore,
+    ITypedConverter<ICurve, RG.Curve> curveConverter
+  )
+  {
+    _settingsStore = settingsStore;
+    _curveConverter = curveConverter;
+  }
+
+  /// <summary>
+  /// Converts a Speckle Region geometry to a Rhino Hatch.
+  /// </summary>
+  /// <param name="target">The Speckle Region geometry to convert.</param>
+  /// <returns>The converted Rhino Hatch.</returns>
+  /// <remarks>⚠️ This conversion does NOT perform scaling.</remarks>
+  public RG.Hatch Convert(SOG.Region target)
+  {
+    List<RG.Curve> rhinoCurves = new() { _curveConverter.Convert(target.boundary) };
+    foreach (var loop in target.innerLoops)
+    {
+      // TODO: add extra checks depending on edge case Regions sent from other apps (e.g. not closed)
```

Comment:
remove this todo if this isn't done in the pr: I wouldn't worry about object validation atm, if there's a bad hatch then let it throw on create below